src/python/pants/backend/python/goals/export.py

```diff
@@ -123,6 +128,30 @@ class ExportPluginOptions:
         removal_hint="Set the `[export].py_resolve_format` option to 'symlinked_immutable_virtualenv'",
     )
 
+    py_editables_in_resolves = StrListOption(
```

Comment:
I went through a lot of different names - I tried putting it on a `PythonSetup(Subsystem)` but that felt weird because this is specifically about export. So, then I moved it here and went with the `py_` prefix similar to `py_resolve_format`. I'm still not entirely happy with it. But, this is the least bad I've come up with so far.

Reply:
How about `py_editable_by_resolve`?

I know it's a list; so plural. But it is used on a "one by one" basis, so this reads better to me. It answers: "Are python distributions to be installed in `editable` mode in this resolve?" for a given resolve. The "editable" thing is not plural in that question, nor is the resolve, but the source for the answer is used for multiple instances of that question.

Reply:
Each dist in the resolve is installed as editable, which is why I was thinking of using `editables` -- each resolve has more than one editable wheel installed.

As I struggle with naming this, one thing I'm worried about is: You can do an editable install of ~anything. If you request an editable install of a git repo, for example, pip will clone it and then you'll be able to see where it is cloned in `pip list` output. So, I want to make it clear that this is only editable for first party code.

That said, Hmm. `by` makes this sound like a map. I could however drop the plural and do `py_editable_in_resolve`. Is that any better?

Reply:
> That said, Hmm. `by` makes this sound like a map. I could however drop the plural and do `py_editable_in_resolve`. Is that any better?

To me, yes. As I see the `editable` as an adjective describing how something(s) are installed it looks weird to pluralize it. Maybe I'm overanalyzing this however.. language is.. rather subjective, perhaps.

Reply:
OK. I just pushed a change with `py_editable_in_resolve`.